tests/test_fx.py

```diff
@@ -226,6 +227,15 @@ def test_normalize():
     clip = audio_normalize(clip)
     assert clip.max_volume() == 1
 
+def test_time_stretch():
+    clip = AudioFileClip("media/crunching.mp3")
+
+    clip1=time_stretch(clip, factor=0.5)
+    assert abs(clip1.duration - 2*clip.duration) < 0.1
```

Comment:
Could you replace this by [`math.isclose`](https://docs.python.org/3/library/math.html#math.isclose) ? If I'm not mistaken, it should be identical.

Reply:
Thanks for your feedback! I made the changes.